src/app.js

```diff
@@ -1 +1,6 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { copyFiles } = require('./helpers/copyFile');
+
+copyFiles('Enter your command ');
```

Comment:
It would be great if i can start typing in the new line
```suggestion
copyFiles('Enter your command: \n');
```

Reply:
**Fixed**